### ISSUE TITLE
[question] Could I continue working on BochsLauncher?

### ISSUE THREAD

**Issue by MVoloshin** [lubomyr/bochs#22]
@lubomyr , hi! Please, tell me, could I continue working on BochsLauncher so it compiles targeting latest Android API? I could also try to bring BochsLauncher into official Bochs repository: https://github.com/bochs-emu/Bochs if maintainers agree. P.S. I hope you're fine

**Comment by lubomyr** [comment]
@MVoloshin , Yes, of course, you can continue to develop and maintain this project. I am fine. Thanks.

**Comment by MVoloshin71** [comment]
@lubomyr , just one more question: how could I properly build BochsLauncher together with Bochs Core (as a single application)?

**Comment by lubomyr** [comment]
@MVoloshin71 ,  Try to run build-all.sh script in bochs/build/android directory. If all build succesfully you will have single apk file with core and launcher

**Comment by MVoloshin** [comment]
@lubomyr , it worked, thx very much. I migrated your project to AndroidX (https://developer.android.com/jetpack/androidx/migrate?hl=ru) library and now it successfully builds for API 33. P.S. Do you know if it's ok to request MANAGE_EXTERNAL_STORAGE on Android 11+? Google Play is very suspicious about that permission, but I don't know any better solutions for Bochs (at least yet)

**Comment by lubomyr** [comment]
@MVoloshin71, No idea, i didn't work with android development for long time.

сб, 4 січ. 2025 р., 13:35 користувач MVoloshin ***@***.***>
пише:
